client/py/aou_workbench_client/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""AoU Workbench Python Client
```

Comment:
Is on the same line as the quotes convention? Or should `AoU Workbench Python Client` move down a line?

Reply:
Yep, the convention is:

```
"""Single-line summary of documentation.

Lots of details.
Lots of details.
Last line of details, closing quotes by themselves.
"""
```

Reply:
Sounds good!